cleanlab/object_detection/summary.py

```diff
@@ -28,14 +28,9 @@
 
 from cleanlab.internal.object_detection_utils import bbox_xyxy_to_xywh
 
-if TYPE_CHECKING:
-    from PIL.Image import Image as Image  # pragma: no cover
-else:
-    Image = TypeVar("Image")
-
 
 def visualize(
-    image: Union[str, Image],
+    image: Union[str, np.ndarray],
```

Comment:
https://github.com/cleanlab/cleanlab/actions/runs/6253132280/job/16977883694
This would cause the plt.imread to fail, I'll try adding a workaround for this